src/app/quiz-toolbar/quiz-toolbar.component.css

```diff
@@ -0,0 +1,6 @@
+.progress-bar {
+    background: url('/src/assets/gokuKamehameha.jpg');
```

Comment:
You traditionally do not want to make an image that is intended to be used as an icon as a background.